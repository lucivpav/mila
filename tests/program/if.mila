var a : integer;
begin
  a := 2;
  if a <> 2 then
    writeln(0);
  writeln(1);
end.
---input---
var a : integer;
begin
  a := 2;
  if a <> 2 then
    writeln(0)
  else
    writeln(1);
end.
---input---
var a : integer;
begin
  a := 2;
  if a = 2 then
    writeln(1)
  else
    writeln(0);
end.
---input---
var a : integer;
begin
  a := 2;
  if a = 2 then
    writeln(1);
  writeln(1);
end.
