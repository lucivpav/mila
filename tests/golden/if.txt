1
0
---output---
1
0
---output---
1
0
---output---
1
1
0
